cylc/flow/network/scan.py

```diff
@@ -326,9 +326,7 @@ async def is_active(flow, is_active):
     except FileNotFoundError:
         _is_active = False
     else:
-        _is_active = any(
-            path.name == WorkflowFiles.Service.CONTACT for path in contents
-        )
+        _is_active = CONTACT in contents
```

Comment:
This suggestion doesn't work, `CONTACT` is a rel path, dir listings are absolute which is why I had used `path.name`.

Reply:
D'oh